### ISSUE TITLE
Updated Environment variables not fed to running container app

### ISSUE THREAD

**Issue by Darren-Vine** [cloudfoundry-community/terraform-provider-cloudfoundry#378]
We have noticed that if we update the environment variables but the docker image has not changed that the docker image does not know about the new environment variable until we manually restage.  

cf env shows that the environment variable has been added, but ssh'ing into the app and displaying the environment variable shows that it doesnt exist.

**Comment by khalilj** [comment]
@Darren-Vine I assume the expected behavior is that CF should perform rolling/blue-green update so the containers are recreated and receive the new environment variables.

**Comment by Darren-Vine** [comment]
Yes,  for BG I would expect it to perform a BG deployment really if that is the only way to get the values passed to the container correctly.  Or to recreate or restage.  Which everyone is the 'correct' way for CF with as little service interruption as possible.
